app/models/batch/tecan_behaviour.rb

```diff
@@ -7,7 +7,10 @@ def generate_tecan_data(target_barcode, override_plate_type = nil)
       'destination' => {}
     }
 
-    requests.includes([{ asset: %i[plate map] }, { target_asset: [:map, :well_attribute, { plate: :barcodes }] }])
+    requests.includes([
+      { asset: [{ plate: [:barcodes, :labware_type] }, :map] },
+      { target_asset: [:map, :well_attribute, { plate: [:barcodes, :labware_type] }] }
+    ])
             .where(state: 'passed')
```

Comment:
Added plate types to the eager loading, although as we cache things on a per plate basis this won't be a huge speed increase.